src/package-lock.json

```diff
@@ -1,8 +1,1498 @@
 {
   "name": "alm-octane-pipeline-start",
   "version": "1.0.2",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```

Comment:
Why are there so many new changes? I would not expect that. Can you please analyze why this happened and limit it?